config/internal/mariadb/deployment.yaml.tmpl

```diff
@@ -26,6 +26,7 @@ spec:
         component: data-science-pipelines
         dspa: {{.Name}}
     spec:
+      serviceAccountName: mariadb-sa-{{.Name}}
```

Comment:
gotta update this as well to match the updated SA name ;)

Reply:
Oops, updated locally and didn't push. Pushed now, should be updated.